docs/get-started/install.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Install
+parent: Get Started
+nav_order: 1
+---
+
+# Install ONNX Runtime
+
+Use this guide to install ONNX Runtime and its dependencies, for your target operating system, hardware, accelerator, and language.
```

Comment:
Would be good to have @snnn help review the install instructions for accuracy

Reply:
Look good to me. The document is correct, but, please think: would it be a bit arrogant if we ask all the onnxruntime users in this world must use en_US as their primary language ? en_US means "English, United States".

Reply:
I'm adding a link checker.